source/Armament.cpp

```diff
@@ -26,17 +26,18 @@ using namespace std;
 
 
 // Add a gun hardpoint (fixed-direction weapon).
-void Armament::AddGunPort(const Point &point, const Outfit *outfit)
+void Armament::AddGunPort(const Point &point, const Angle &angle, bool isParallel, const Outfit *outfit)
 {
-	hardpoints.emplace_back(point, false, outfit);
+	hardpoints.emplace_back(point, angle, false, isParallel, outfit);
 }
 
 
 
 // Add a turret hardpoint (omnidirectional weapon).
 void Armament::AddTurret(const Point &point, const Outfit *outfit)
 {
-	hardpoints.emplace_back(point, true, outfit);
+	bool isParallel = false;
+	hardpoints.emplace_back(point, Angle(0.), true, isParallel, outfit);
```

Comment:
I'd just do this, as why would you create an isParallel variable so that we know what is being set as false but not have a variable for whatever is set as true? If someone wants to know what these booleans are for, I'd say to just look at Hardpoint's constructor(s).
```suggestion
	hardpoints.emplace_back(point, Angle(0.), true, false, outfit);
```